templates/index.html

```diff
@@ -55,8 +54,9 @@
 
     <p class="panel-heading">Current Issue</p>
 
-    {% if current_issue.cover_image %}
-      <a href="{{ url('current') }}"><div id="current-cover">{{ thumbnail(current_issue.cover_image, "320x464") }}</div></a>
+   {% if cover %}
+      <a href="{{ url('current') }}"><div id="current-cover">{{ thumbnail(cover.cover_image.url, "320x464") }}</div></a>
```

Comment:
The function `thumbnail` expects an instance of image field instead of a url as its first argument.
i.e.
`cover.cover_image.url`
should be:
`cover.cover_image`